src/EnergyPlus/Furnaces.cc

```diff
@@ -12099,6 +12099,7 @@ namespace Furnaces {
         if (state.dataFurnaces->Furnace(FurnaceNum).OpMode == ContFanCycCoil) state.dataHVACGlobal->OnOffFanPartLoadFraction = 1.0;
 
         auto &outNode = state.dataLoopNodes->Node(OutletNode);
+        AirMassFlow = outNode.MassFlowRate;
```

Comment:
Up at 11520 is this line that can be deleted:

    AirMassFlow = state.dataLoopNodes->Node(InletNode).MassFlowRate;

CppCheck on this branch shows this:

    [src/EnergyPlus/Furnaces.cc:12102]:(style),[redundantAssignment],Variable 'AirMassFlow' is reassigned a value before the old one has been used.